### DISCUSSION TITLE
How to prevent <a> tags with the attribute target=_blank from opening new pages?

### DISCUSSION THREAD

**Discussion by manyougz** [tauri-apps/tauri#11580] [Q&A] [answered]
Some websites have `<a>` tags with the attribute `target=_blank` that automatically open new pages when clicked. However, this causes them to leave the Tauri Webview Window. How can I ensure they open within the Tauri Webview Window instead of launching a new page in an external browser?

**Comment by FabianLars** [comment, marked as answer]
currently this behavior can only be disabled by removing the `shell` plugin. but even then it may open a new uncontrolled app window instead of launching it in the same window.

**Reply by manyougz** [reply]
Hopefully, there will be a better solution in the future.
